backend/corpora/common/entities/dataset.py

```diff
@@ -0,0 +1,28 @@
+import typing
+
+from .entity import Entity
+from ..corpora_orm import DbDataset
+
+
+class Dataset(Entity):
+    tables = [
+        DbDataset,
+    ]
+
+    def __init__(self, db_obj: DbDataset):
+        self.db_obj = db_obj
+
+    @classmethod
+    def _query(cls, key: str) -> typing.List["Dataset"]:
```

Comment:
Add docstring? I find it good practice to have it even for "self-explanatory" functions because it forces that the implementation matches what we meant to do written in plain English.

Reply:
fixed